networking/network_policy/creating-network-policy.adoc

```diff
@@ -12,7 +17,13 @@ include::modules/nw-networkpolicy-create.adoc[leveloffset=+1]
 
 include::modules/nw-networkpolicy-object.adoc[leveloffset=+1]
 
+ifdef::openshift-rosa,openshift-dedicated[]
+include::modules/ocm-nw-networkpolicy-create.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::openshift-enterprise[]
```

Comment:
You could consider using `ifndef::openshift-dedicated,openshift-rosa[]` instead, in case the enclosed content is required for other distros, such as OKD.